deepchecks/nlp/utils/text_properties.py

```diff
@@ -194,6 +196,39 @@ def formality(raw_text: Sequence[str], device: Optional[int] = None) -> List[flo
     return [x['score'] if x['label'] == 'formal' else 1 - x['score'] for x in classifier(raw_text)]
 
 
+def lexical_density(raw_text: Sequence[str]) -> List[str]:
+    """Return a list of floats of lexical density per text sample.
+
+    Lexical density is the percentage of unique words in a given text. For more
+    information: https://en.wikipedia.org/wiki/Lexical_density
+    """
+    nltk_download('punkt', quiet=True)
```

Comment:
add an if statement here so that if download fails, you return raise a warning and return np.nan for all samples.
Same for next check

Reply:
Done